src/main/java/com/sda/dao/CategoryDao.java

```diff
@@ -0,0 +1,17 @@
+package com.sda.dao;
+
+import com.sda.entity.Category;
+import org.hibernate.Session;
+
+public interface CategoryDao {
```

Comment:
Maybe it is better to create some GeneralDao class since your following methods are standard CRUD operations